app/routes/event.tsx

```diff
@@ -0,0 +1,44 @@
+import { LoaderFunction, redirect, LinksFunction } from '@remix-run/node';
+import styles from 'react-datepicker/dist/react-datepicker.css';
+import { Outlet, useCatch, Link } from '@remix-run/react';
+import { CalendarIcon } from '@heroicons/react/24/outline';
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const url = new URL(request.url);
+  if (url.pathname === '/event') {
+    return redirect('/');
+  }
```

Comment:
Why are we doing this here?

Reply:
if user tries to go to `/events` page, the we redirect them to `/` page